src/components/Books.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable react/prop-types */
```

Comment:
Rather than disable the linter, strive to fix them as this makes you more accustomed to the style guide of the tech.